modules/vfh_decision/vfh_decision.py

```diff
@@ -0,0 +1,92 @@
+"""
+Processes VFH data to compute optimal steering directions.
+"""
+
+from modules import polar_obstacle_density
+
+
+class VFHDecision:
+    """
+    Computes steering direction from polar obstacle densities.
+    """
+
+    def __init__(
+        self,
+        density_threshold: float,
+        min_consec_sectors: int,
+        wide_valley_threshold: int,
+        target_angle: float,
+    ) -> None:
+        """
+        Initializes VFHDecision with density threshold and valley parameters.
+
+        :param density_threshold: The threshold for identifying candidate valleys.
+        :param min_consec_sectors: Minimum number of consecutive sectors for a valid valley.
+        :param wide_valley_threshold: Number of sectors distinguishing wide valleys from narrow ones.
```

Comment:
why do we define a wide valley threshold? is a narrow valley not wide enough? is this the minimum narrow valleys required for the drone to fly down the valley? if so, can the description make this more clear?

Reply:
it's for a refinement outlined in the research paper. Technically we can go lower than the wide_threshold, but that would require us to slow down our flight_speed (formula given in paper, relates to maximal steering speed) so could implement that in the future.

Reply:
(also its just a bit safer, until we concretely decide on a safe threshold based on tests)